Themes/default/scripts/script.js

```diff
@@ -326,6 +326,11 @@ function reqOverlayDiv(desktopURL, sHeader, sIcon)
 		},
 		error: function (xhr, textStatus, errorThrown) {
 			oPopup_body.html(textStatus);
+		},
+		statusCode: {
+			500: function() {
+				oPopup_body.html(banned_text);
```

Comment:
It would be good to do this: 
```
if (sHeader == 'Login')
	oPopup_body.html(banned_text);
else
	oPopup_body.html('500 Internal Server Error');
```

Then if the Ajax call ever receives a 500 status code response for something besides the login form, it won't display the banned message.